js/utils.js

```diff
@@ -57,7 +72,10 @@ export function clearEvents() {
     multipleEventElement.remove()
   );
 }
-
+export function closeEventDetailsModal() {
+  const eventDetailsModal = document.getElementById('eventDetailsModal');
+  eventDetailsModal.style.display = 'none';
+}
```

Comment:
this function should not be part of utils, closing functionality should come from modal logic